src/AdTypes.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\fb_instant_articles_display\DrupalInstantArticleDisplay.
+ */
+
+namespace Drupal\fb_instant_articles;
+
+abstract class AdTypes {
```

Comment:
So I'm asking because the use statement and class calls in the form seem like an awful lot for an array of four options. Maybe we just make a function in the module that defines the array? (I know we can define an array in a constant pre-5.6 using serialize, but that doesn't help you access it in the class and settings form whereas a simple function would). 

Something like this:
```
function fb_instant_articles_get_ad_types() {
  return array(
    'none' => 'None',
    'fban' => 'Facebook Audience Network',
    'source_url => 'Source URL',
    'embed_code' => 'Embed Code',
  );
}
```